### ISSUE TITLE
Landscape mode in BottomSheet fails accessibility test

### ISSUE THREAD

**Issue by proohit** [FormidableLabs/react-native-ama#223]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Bug report

```markdown
Currently, the BottomSheet component supports landscape mode only in Android. I assume, this is because `Modal#supportedOrientations` is set to `portrait` by default but I am not sure 100%.

Supporting landscape mode is be crucial for accessibility. Therefore I think supporting this property, or at least defaulting it to `["portrait", "landscape"]` would be good.

Thanks in advance.
```
